packages/grid/x-data-grid-generator/src/renderer/renderProgress.tsx

```diff
@@ -1,20 +1,37 @@
 import * as React from 'react';
 import clsx from 'clsx';
 import { GridRenderCellParams } from '@mui/x-data-grid-premium';
-import { styled } from '@mui/material/styles';
+import { styled, alpha } from '@mui/material/styles';
 
 interface ProgressBarProps {
   value: number;
 }
 
-const Root = styled('div')(({ theme }) => ({
-  border: `1px solid ${(theme.vars || theme).palette.divider}`,
+const Root = styled('div')({
   position: 'relative',
   overflow: 'hidden',
   width: '100%',
   height: 26,
-  borderRadius: 2,
-}));
+  borderRadius: 3,
+  '&.low': {
+    backgroundColor: alpha('#FF505F', 0.1),
+    ' .progress-bar': {
```

Comment:
Thanks!
I wonder what would be the idiomatic Emotion approach here to avoid global class names?
Styling background colors of `Root` and `Bar` separately in their own `styled()`?

Reply:
Pushed a commit regarding this. Let me know what you think 😄